public/app/plugins/panel/table/module.tsx

```diff
@@ -75,10 +82,52 @@ export const plugin = new PanelPlugin<PanelOptions, PanelFieldConfig>(TablePanel
     },
   })
   .setPanelOptions((builder) => {
-    builder.addBooleanSwitch({
-      path: 'showHeader',
-      name: 'Show header',
-      description: "To display table's header or not to display",
-      defaultValue: defaultPanelOptions.showHeader,
-    });
+    builder
+      .addBooleanSwitch({
+        path: 'showHeader',
+        name: 'Show header',
+        description: "To display table's header or not to display",
+        defaultValue: defaultPanelOptions.showHeader,
+      })
+      .addBooleanSwitch({
+        path: 'showFooter',
+        name: 'Show Footer',
+        description: "To display table's footer or not to display",
+        defaultValue: defaultPanelOptions.showFooter,
+      })
+      .addCustomEditor({
+        id: 'footer.reducer',
+        path: 'footer.reducer',
+        name: 'Calculation',
+        description: 'Choose a reducer function / calculation',
+        editor: standardEditorsRegistry.get('stats-picker').editor as any,
+        defaultValue: [ReducerID.sum],
+        showIf: (cfg) => cfg.showFooter,
+      })
+      .addMultiSelect({
+        path: 'footer.fields',
+        name: 'Fields',
+        description: 'Select the fields that should be calculated',
+        settings: {
+          allowCustomValue: false,
+          options: [],
+          placeholder: 'All Numeric Fields',
+          getOptions: async (context: FieldOverrideContext) => {
```

Comment:
So we have a addFieldNamePicker but I see it's missing a multi-select option, wonder if we should add that to that option or have some other built in multi fields selector.  throughts @ryantxu ?